app/components/graph-editor/graph-editor.component.ts

```diff
@@ -334,6 +334,18 @@ export class GraphEditorComponent implements AfterViewInit {
         this.redrawDelegate();
     }
 
+    copy(): void {
+        console.log("Editor copy");
+    }
+
+    cut(): void {
+        console.log("Editor cut");
+    }
+
+    paste(): void {
+        console.log("Editor paste");
+    }
```

Comment:
I still think it probably makes sense for the graph editor to be in charge of cut, copy, paste itself.  The main component is more of a director class and the nitty gritty implementation of stuff should be outsourced, in this case either to graph editor itself or something else.